### PR TITLE
Added support for symfony/console 6.0

### PR DESCRIPTION
**Pull request by acelaya**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | yes
| RFC           | no
| QA            | no

### Description

This PR adds support for symfony/console 6, as it's a pretty popular component, and the update is pretty much backwards compatible.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
         "psr/http-message-implementation": "^1.0",
         "psr/http-server-handler": "^1.0",
         "psr/log": "^1.0",
-        "symfony/console": "^4.1 || ^5.0",
+        "symfony/console": "^4.1 || ^5.0 || ^6.0",
```
Comment (Ocramius) [inline composer.json]: Can we drop 4.x while at it?

**Review by Ocramius — CHANGES_REQUESTED** [review]

**Comment by Ocramius** [comment]
Question: does this result in a downstream BC break, due to inheritance?

File: composer.json (hunk 1 above)
Comment (acelaya) [reply composer.json]: Sure!

**Comment by acelaya** [comment]
I think I don't follow. What do you mean?

**Comment by weierophinney** [comment]
> Question: does this result in a downstream BC break, due to inheritance?

It shouldn't. I've used symfony/console 6 with laminas-cli versions, and had no issues with commands. The issues will be if a user is writing commands that rely on portions of the symfony/console v4 API that are not present in later versions, or changed in later versions. Since they're very conservative about changing the inheritance API, in practice, I've not seen any cases where such breakage can occur.

**Review by Ocramius — CHANGES_REQUESTED** [review]
`composer.lock` still requires adjusting here

**Comment by weierophinney** [comment]
@Ocramius I updated the `composer.lock`, but the fork was locked and I could not push back to it. Hoping that the branch builds correctly; if not, I'll open another PR with a patch to resolve.

(Bonus points: bumped to laminas-coding-standard 2.3, which means I could get rid of the "ignore-platform-reqs" flag!)

**Comment by acelaya** [comment]
I was about to update the composer.lock, but I guess it's not needed anymore 😅

**Pull request merged**